### PR TITLE
make questionable choices

### PR DESCRIPTION
**Pull request by lcruz4**

### CODE REVIEW

File: src/server/api/routers/posts.ts [hunk 1]
```diff
@@ -33,6 +33,28 @@ const ratelimit = new Ratelimit({
...
+    const posts = await ctx.prisma.post.findMany({
+      where: { authorId: input },
+    });
+    if (!posts[0]) {
+      throw new TRPCError({
+        code: "NOT_FOUND",
+        message: "No posts found",
+      });
+    }
+    const author = (await clerkClient.users.getUserList({ username: [posts[0].authorId] }))[0];
```
Comment (lcruz4) [inline src/server/api/routers/posts.ts:46]: can't use getUser anymore because that requires the id :(

**Review by dmccloud — COMMENTED** [review]
image
ya shit brokez

File: src/server/api/routers/posts.ts [hunk 2]
```diff
@@ -41,25 +63,24 @@ export const postsRouter = createTRPCRouter({
 
     const users = (
       await clerkClient.users.getUserList({
-        userId: posts.map((post) => post.authorId) as string[],
+        username: posts.map((post) => post.authorId) as string[],
         limit: 100,
       })
     ).map(fulterUserForClient);
 
+    console.log(users)
```
Comment (dmccloud) [inline src/server/api/routers/posts.ts:71]: Is this my PR?

**Review by dmccloud — COMMENTED** [review]
Looks good, but I think it brokeded the post view somehow? Havent had a chance to look too closely. I will check laterz

File: src/server/api/routers/posts.ts (hunk 2 above)
Comment (lcruz4) [reply src/server/api/routers/posts.ts:71]: this is why i need precommits!!!!!!!!

**Review by dmccloud — APPROVED** [review]
V wow much indifferencez

**Pull request merged**